app/controllers/reservations_controller.rb

```diff
@@ -2,18 +2,23 @@ class ReservationsController < ApplicationController
   before_action :authenticate_user!
 
   def index
```

Comment:
Looking at the current content of `routes.rb` I see that you use this method for two paths:

```
                     seance_reservations GET    /seances/:seance_id/reservations(.:format)                                                        reservations#index
                            reservations GET    /reservations(.:format)                                                                           reservations#index
```

Do you awarely want to use the same method for both paths? If yes, then you probably should take care of scoping `reservations` to a chosen seance in one case :wink:

Reply:
It is a leftover from previous version, where I only listed each seance reservations. I will remove it:)